src/helpers/Extensions/withBlockSchemaEnhancer.js

```diff
@@ -38,6 +38,20 @@ function getBlocksConfig(props) {
   return blocks?.blocksConfig;
 }
 
+export function getVariations({ variations, schema, formData, properties }) {
+  return variations.filter((variation) => {
+    if (variation.enabled !== undefined) {
+      if (typeof variation.enabled === 'function') {
+        const variationFunc = variation.enabled;
+        variationFunc({ schema, formData, properties });
```

Comment:
@sneridagh Can this function also look things up from the redux state (i.e. current subsite theme, etc)?